deepchecks/tabular/checks/model_evaluation/performance_bias.py

```diff
@@ -0,0 +1,549 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""The performance_disparity_report check module."""
+import itertools
+from typing import Callable, Tuple, Union
+
+import numpy as np
+import pandas as pd
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+from deepchecks.core import CheckResult, ConditionCategory, ConditionResult
+from deepchecks.core.checks import DatasetKind
+from deepchecks.core.errors import DeepchecksProcessError, DeepchecksValueError
+from deepchecks.tabular import Context, SingleDatasetCheck
+from deepchecks.utils.performance.partition import partition_column
+from deepchecks.utils.typing import Hashable
+
+
+class PerformanceBias(SingleDatasetCheck):
+    """
+    Check for performance disparities across subgroups, while optionally accounting for a control variable.
```

Comment:
First line is very vague and I wouldn't understand what the check does by reading it.